DevCycle/DVCClient.swift

```diff
@@ -81,7 +81,7 @@ public class DVCClient {
             return
         }
         self.service = service
-        self.service?.getConfig(user: user, enableEdgeDB: self.enableEdgeDB, completion: { [weak self] config, error in
+        self.service?.getConfig(user: user, enableEdgeDB: self.enableEdgeDB, extraParams: nil, completion: { [weak self] config, error in
```

Comment:
didn't we call this `requestParams` on the JS SDK

Reply:
it's called `requestParams` in the `RequestConsolidator`'s `queue` method but `extraParams` in the `getConfigJson` method. I figured `extraParams` made more sense here cause  `user`  and `enabledEdgeDB` are both required params but the sse stuff may or may not be on the request. I can change it if you'd like. Just let me know